Alloy/commands/compile/index.js

```diff
@@ -248,6 +253,59 @@ module.exports = function(args, program) {
 			}
 		});
 	}
+
+	// copy the platform and theme platform directories
```

Comment:
We **must** and **can** make this change non-breaking. If we don't, people will loose their platform and i18n files at project-level.

Simply after this line add:

```
	if (fs.existsSync(path.join(paths.project, 'platform')) && !fs.existsSync(path.join(paths.app, 'platform'))) {
		U.die('Please move the /platform folder to /app/platform for Alloy 1.8.0 or later.');
	}
```